### DISCUSSION TITLE
欢迎来到 SakuraFrp 用户讨论，请在发言前阅读此帖

### DISCUSSION THREAD

**Discussion by fakeboboliu** [natfrp/wiki#53] [公告]
## 👋 欢迎！
  这里是 SakuraFrp 帮助文档的讨论区，我们希望这里为用户社区提供一个更加直接的交流渠道。
  这里不会限制用户组，但是我们希望通过 GitHub 平台的选择限制会在这里发言的用户——那些知道自己在做什么的人。

  我们希望你在这里：
  * 与其他社区成员交流使用相关话题 （在 [通常话题](https://github.com/natfrp/wiki/discussions/categories/%E9%80%9A%E5%B8%B8%E8%AF%9D%E9%A2%98) 分类）
  * 提出使用的问题（回不回答看其他人的心情） （在 [使用问题](https://github.com/natfrp/wiki/discussions/categories/%E4%BD%BF%E7%94%A8%E9%97%AE%E9%A2%980) 分类）
  * 提出对我们服务、软件的意见和想法 （在 [想法](https://github.com/natfrp/wiki/discussions/categories/%E6%83%B3%E6%B3%95) 分类）

  我们希望我们可以在这里：
  * 看到您的意见，以此改进我们的服务
  * 与非 VIP 用户直接交流
  * 在某些时候，可能发布一些投票作为服务改进的社区意见

  **如果你在这里发出了下面类型的提问，我们可能会在这个项目中拉黑你**：
  * 文档已覆盖
  * 没头没尾
  * 其他任何我们认为不属于一个极客会提出来的问题

  为了服务的稳定，我们不希望在这里看到：
  * 节点 IP 或 节点域名
  * 您的访问密钥 （token）

  对于希望官方回答的使用问题，请参考 联系我们 页面，使用问题分类在预期中将有极少官方人员出没。

**Comment by railgun19982** [comment]
草，什么返璞归真（话说为什么直接不跑去用akk的“论坛 via github”）

**Reply by SanaeAkahane** [reply]
《他甚至在帮助文档仓库下创建了个论坛》

**Reply by XJB3711** [reply]
.....

**Reply by fakeboboliu** [reply]
又不是不能用™

 - [x] 不需要维护
 - [x] 不会被打
 - [x] 能防麦片哥
 - [x] 使用门槛低

**Comment by xiongxiongya** [comment]
离谱

**Comment by choneas** [comment]
提出沙雕问题对吧?[doge]

**Comment by keptupp** [comment]
为啥我的隧道启动成功，显示也是在线的状态，但是ping这个端口号ping不通，只能ping通默认端口

**Reply by keptupp** [reply]
说是请求找不到主机

**Comment by 13234awd** [comment]
我的隧道开好了一切正常，和朋友玩了大概两个小时我的世界，然后忽然断了，期间没有改变网络或者模组等，突然断了，他在想进我房间一直显示无效对话，尝试重开等方法依然无法解决，我不知道问题出现在哪里
